pkg/apis/core/v1alpha1/portforward_types.go

```diff
@@ -169,6 +176,12 @@ func (in *PortForward) Validate(_ context.Context) field.ErrorList {
 	return fieldErrors
 }
 
+func (in *PortForward) Default() {
```

Comment:
i would assign this to the Defaulter interface, just to make sure they didn't add extra methods or smth

var _ resourcestrategy.Defaulter = &PortForward{}